src/scripts/demo.mjs

```diff
@@ -107,6 +158,7 @@ const MOCK_COMFY = (function () {
 	};
 
 	const allUsers = [broadcaster, ...mods, broadcaster, ...viewers, broadcaster];
+	USERS = [broadcaster, ...mods, ...viewers];
```

Comment:
What's the reason for these 2 arrays?
Can the first one be reused? (setting `let allUser = [];` at the top of the file and assigning it here).
Why is `broadcaster` used three times in the `allUser` array?

Reply:
Great question - agreed that the duplication is kind of confusing. The reason `allUsers` has several `broadcaster` instances is so that the whole array can be weighted to pick the broadcaster three times more often than any of the fake mods/viewers. However, I think we could achieve a similar-ish effect by just generating fewer fake mods/viewers — which might be worth it anyways. I'm no sure demo really needs 20 "chatters."